src/components/product-gallery/ProductGalleryUI.test.tsx

```diff
@@ -377,14 +451,103 @@ describe(ProductGalleryUI.name, () => {
 
     // Act
     render(
-      <TestContextWrapper>
-        <ProductGalleryUI {...baseProps} debouncedSearchTerm="search term" />
-      </TestContextWrapper>,
+      <QueryClientProvider client={queryClient}>
+        <AppServicesProvider appServices={appServices}>
+          <TestContextWrapper>
+            <ProductGalleryUI
+              {...baseProps}
+              debouncedSearchTerm="search term"
+            />
+          </TestContextWrapper>
+        </AppServicesProvider>
+      </QueryClientProvider>,
     );
 
     // Assert
     screen.getByText("products_gallery_search_no_results_message");
   });
+
+  it("should be a store selected with active style", () => {
+    // Arrange
+    const baseProps = createBaseProps();
+
+    // Act
+    render(
+      <QueryClientProvider client={queryClient}>
+        <AppServicesProvider appServices={appServices}>
+          <TestContextWrapper>
+            <ProductGalleryUI {...baseProps} storeSelected="MercadoShops" />
+          </TestContextWrapper>
+        </AppServicesProvider>
+      </QueryClientProvider>,
+    );
+
+    // Assert
+    const storeItem = screen.getByText("MercadoShops");
+    expect(storeItem.style.color).toEqual("rgb(38, 38, 38)");
+  });
+
+  it("should be a store with no active style", () => {
```

Comment:
En este caso ningun store estaria seleccionado, no? es decir, todos tendrian el "no active style"?

Reply:
that right